### PR TITLE
tag ClimaCore 0.7

### PR DESCRIPTION
**Pull request by simonbyrne**

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]
Much better than mine, thank you 😂

**Comment by simonbyrne** [comment]
bors try

**Comment by simonbyrne** [comment]
bors try

File: perf/allocs_per_case.jl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Launch with `julia --project --track-allocation=user`
-import Pkg
-Pkg.develop(path = abspath(joinpath(@__DIR__, "..")))
+EXAMPLE_DIR = joinpath(dirname(@__DIR__), "examples")
+EXAMPLE_DIR in LOAD_PATH || pushfirst!(LOAD_PATH, EXAMPLE_DIR)
```
Comment (charleskawczynski) [inline perf/allocs_per_case.jl:3]: I think there's an issue with using `LOAD_PATH` when we have ClimaCore as an env dep. IIUC, we need to use `Pkg.develop` in this case. Is there a reason we can't stick with what we had?

**Comment by simonbyrne** [comment]
bors try

File: perf/allocs_per_case.jl (hunk 1 above)
Comment (simonbyrne) [reply perf/allocs_per_case.jl:3]: The problem was it couldn't resolve the ClimaCorePlots correctly. So if this doesn't work, we will have to check in the Manifest.toml

**Comment by simonbyrne** [comment]
bors try

File: perf/allocs_per_case.jl (hunk 1 above)
Comment (simonbyrne) [reply perf/allocs_per_case.jl:3]: @charleskawczynski that seemed to work: can you check the output and see how it looks?

Comment (charleskawczynski) [reply perf/allocs_per_case.jl:3]: Ah, I see. Yeah, this config looks fine, and the allocations monitoring looks fine, too.

**Pull request merged**